src/bigint.rs

```diff
@@ -1578,6 +1649,49 @@ impl BigInt {
 
     /// Creates and initializes a `BigInt`.
     ///
+    /// The bytes are in big-endian byte order
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num::bigint::{BigInt, Sign};
+    ///
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "A".as_bytes()),
+    ///            BigInt::parse_bytes("65".as_bytes(), 10).unwrap());
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "AA".as_bytes()),
+    ///            BigInt::parse_bytes("16705".as_bytes(), 10).unwrap());
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "AB".as_bytes()),
+    ///            BigInt::parse_bytes("16706".as_bytes(), 10).unwrap());
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "Hello world!".as_bytes()),
+    ///            BigInt::parse_bytes("22405534230753963835153736737".as_bytes(), 10).unwrap());
+    /// ```
+    #[inline]
+    pub fn from_bytes_be(sign: Sign, bytes: &[u8]) -> BigInt {
+        BigInt::from_biguint(sign, BigUint::from_bytes_be(bytes))
+    }
+
+    /// Creates and initializes a `BigInt`.
+    ///
+    /// The bytes are in little-endian byte order
+    #[inline]
+    pub fn from_bytes_le(sign: Sign, bytes: &[u8]) -> BigInt {
+        BigInt::from_biguint(sign, BigUint::from_bytes_le(bytes))
+    }
+
+    /// Returns the byte representation of the `BigInt` in little-endian byte order
```

Comment:
Sounds reasonable. Could you add this to the documentation?

Hm, maybe it could return `(Sign, Vec<u8>)` to be a true inverse to `from_bytes_le`. One only needs to write an additional `.1` to access the `Vec<u8>` itself.